src/components/hoc/withRestrictedAccess.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/display-name */
+import React from 'react';
+import { Redirect } from 'react-router';
+import { isLoggedIn } from '../../utils/handlers/authenticationHandlers';
+
+const withRestrictedAccess = (BaseComponent, onlyForAdmin = false, user) => props => {
+  if (!isLoggedIn()) {
+    return <Redirect to="/login" />;
+  }
+
+ let isAdmin;
+
+ if (user) isAdmin = user.isAdmin;
+
+  if (onlyForAdmin && !isAdmin) {
```

Comment:
I think there is extra space/indentation before the `if ` statement

Reply:
Done ;D